modepy/shapes.py

```diff
@@ -199,76 +227,51 @@
 @dataclass(frozen=True)
 class Shape:
     """
-    .. attribute :: dims
+    .. attribute :: dim
     .. attribute :: nfaces
     .. attribute :: nvertices
     """
-    dims: int
+    dim: int
 
 
 @singledispatch
-def get_unit_vertices(shape: Shape):
+def biunit_vertices_for_shape(shape: Shape):
     """
-    :returns: an :class:`~numpy.ndarray` of shape `(nvertices, dims)`.
+    :returns: a :class:`~numpy.ndarray` of shape `(dim, nvertices)`.
```

Comment:
```suggestion
    :returns: an :class:`~numpy.ndarray` of shape `(dim, nvertices)`.
```
? Since it's just going to show `ndarray`?